lib/generators/engine_cart/install_generator.rb

```diff
@@ -52,8 +52,7 @@ def add_gemfile_include
   # the below comes from engine_cart, a gem used to test this Rails engine gem in the context of a Rails app
   file = File.expand_path("Gemfile", ENV['ENGINE_CART_DESTINATION'] || ENV['RAILS_ROOT'] || File.expand_path("../spec/internal", __FILE__))
   if File.exist?(file)
-    puts "Loading \#{file} ..." if $DEBUG # `ruby -d` or `bundle -v`
-    instance_eval File.read(file)
+    eval_gemfile file
   else
     # we get here when we haven't yet generated the testing app via engine_cart
     gem 'rails', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
```

Comment:
I think a Rake task is a good idea. It could be a rake task that literally just output the code for you to copy and paste into your Gemfile, and the README directs you to the rake task. Or, more complicatedly and error-prone, it could be a rake task that actually tries to insert into the Gemfile -- and actually tries to replace any previous-version's insert with the current version's insert, perhaps using some comment markers. 

Even just a rake task that outputs the thing for you to manually cut and paste into the gemfile, mentioned in the README, would actually be pretty valuable. This is also something that has ended up taking me non-trivial time to debug, when I go to an old project using an old version of EC and try to update it to a new one, and don't realize I need to figure out and paste new Gemfile source.